### DISCUSSION TITLE
Delay node in comfyui to control CPU usage

### DISCUSSION THREAD

**Discussion by MushieKings** [toyxyz/ComfyUI_toyxyz_test_nodes#28] [Ideas]
I did notice running smaller images at a faster rate really chews up the cpu usage. A simple delay node that can do fractions of a second would be great for fixing this. I sort of made one with code I seen on reddit lol but would be nice to have one built into your node.

Here's the code anyways if anyone's interested
[Latent_delay.txt](https://github.com/toyxyz/ComfyUI_toyxyz_test_nodes/files/13795393/Latent_delay.txt)
save the txt file as .py in custom node folder

**Comment by toyxyz** [comment]
Thank you!
